src/logger.ts

```diff
@@ -0,0 +1,9 @@
+import { createLogger } from "ch-logging";
+import { APP_NAME } from "./app";
+import { LOG_LEVEL } from "./properties";
+
+const logger = createLogger(APP_NAME);
+
+logger.info(`env.LOG_LEVEL set to ${LOG_LEVEL}`);
```

Comment:
Guessing if log level is ever set lower (or is it higher?) than INFO - e.g. ERROR - this line of output will never appear.

Reply:
yes, info is highest we can go without using error

Reply:
there is also WARN but doesn't seem right to use that here

Reply:
forget that, warn is not implemented in this logger library

Reply:
I did wonder about whether this line should be output as a console.log message if the intention is to know what the log level is before any logging is done by the framework. Not sure that's right either though...

Reply:
that should work, i can change it